### ISSUE TITLE
Use already available roles?

### ISSUE THREAD

**Issue by ypid** [openedx-unsupported/configuration#2715]
I just looked over your playbooks and roles shortly and just wanted to drop a hint that the http://debops.org/ has of few roles which seem to do similar things than your roles.

The roles from DebOps should be flexible enough to using them in your playsbooks.

Example: [MariaDB](https://github.com/edx/configuration/blob/master/playbooks/roles/mariadb/tasks/main.yml) -> https://github.com/debops/ansible-mariadb_server

**Comment by fredsmith** [comment]
Unfortunately, most of the roles on debops and other providers on ansible-galaxy don't fit what we need when we're building a playbook.

MariaDB is an odd role because we don't actually use it here at EdX.org, but it was contributed by a community member and we decided to bring it into our repo so others could use it. 

I scanned through the debops library and the only other overlap our roles have with theirs is nginx.  EdX's nginx role is highly customized for our service infrastructure, and it would be difficult to make debops/ansible-nginx work like ours.
